### PR TITLE
[15.0][MIG]website_sale_attribute_filter_form_submit: Migration to 15.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Comment by bizzappdev** [comment]
ready for review

**Review by miguel-S73 — APPROVED** [review]

File: website_sale_attribute_filter_form_submit/static/src/js/website_sale.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ odoo.define(
     function (require) {
         "use strict";
 
+        require("website_sale.website_sale");
```
Comment (chienandalu) [inline website_sale_attribute_filter_form_submit/static/src/js/website_sale.js:8]: Is this needed?

**Review by chienandalu — APPROVED** [review]

**Pull request merged**